modules/go-settings-for-environment-labels-and-annotations.adoc

```diff
@@ -0,0 +1,75 @@
+// Module included in the following assemblies:
+//
+// * /gitops/health-information-for-resources-deployment.adoc
+
+:_content-type: REFERENCE
+[id='go-settings-for-environment-labels-and-annotations_{context}']             
+= Settings for environment labels and annotations
+
+This section provides reference settings for environment labels and annotations required to display an environment application in the *Environments* page, in the *Developer* perspective of the {product-title} web console.
+
+[discrete]
+== Environment labels
+
+The environment application manifest must contain `labels.openshift.gitops/environment` and `destination.namespace` fields. You must set identical values for the `<environment-name>` variable and the name of the environment application manifest.
```

Comment:
Content in brackets should use underscores, as per https://redhat-documentation.github.io/supplementary-style-guide/#user-replaced-values

```suggestion
The environment application manifest must contain `labels.openshift.gitops/environment` and `destination.namespace` fields. You must set identical values for the `<environment_name>` variable and the name of the environment application manifest.
```